### ISSUE TITLE
Copy Blobs between blobstores

### ISSUE THREAD

**Issue by AliveDevil** [owncloud/ocis#9301]
# Description
Currently there is no way of migrating one OCIS installation (with all Spaces included) to another, with a different backend (POSIX -> S3NG, or vice versa).

## User Stories

* > As a sys-admin, I want to be able to copy spaces verbatim, so that I can migrate data to other storage backends.

## Value

- Ease migration between storage backends

## Acceptance Criteria

- Migration cli command that copies all blobs from one blobstore into another

## Definition of ready
- [ ] Everybody needs to understand the value written in the user story
- [ ] Acceptance criteria have to be defined
- [ ] All dependencies of the user story need to be identified
- [ ] Feature should be seen from an end user perspective
- [ ] Story has to be estimated
- [ ] Story points need to be less than 20

## Definition of done
- Functional requirements
  - [ ] Functionality described in the user story works
  - [ ] Acceptance criteria are fulfilled
- Quality
  - [ ] Code review happened
  - [ ] CI is green (that includes new and existing automated tests)
  - [ ] Critical code received unit tests by the developer
- Non-functional requirements
  - [ ] No sonar cloud issues
- Configuration changes
  - [ ] The next branch of the ocis charts is compatible

**Comment by micbar** [comment]
> But we could build a similar command that migrates blobs from one store to another.

Yes. That is what i meant. But you already did some ground work in reva to export some props to get that done.

**Comment by kobergj** [comment]
Sounds good :+1: 

So the action item would be to create a cli command that copies all blobs from one blobstore into another. Using the new `List` method on the source blobstore should make this a straightforward task. We will not validate anything, just copy the blobs into the new store.

**Comment by AliveDevil** [comment]
That would be fine by me, and even more premium than the initial option of just keeping the ids and copying data manually.

**Comment by AliveDevil** [comment]
Updated entry post.
Anything to amend?

**Comment by micbar** [comment]
@tbsbdr @dragotin we need to squeeze that in.

**Comment by micbar** [comment]
Added to backlog. Candidate for next sprint.

**Comment by butonic** [comment]
I am reading this as 'only copy blobs between blobstores' not anything related to metadata, right? The latter would require a `Dump` and `Load` mechanism for node metadata.
